qemu/tests/cfg/netuser_buffer_test.cfg

```diff
@@ -13,6 +13,6 @@
     mtu_value = 12000
     set_mtu_cmd = "ip link set %s mtu ${mtu_value}"
     check_mtu_cmd = "ip link show dev %s"
-    exp_path = netuser/rhel8/exp.c
-    RHEL.9:
-        exp_path = netuser/rhel9/exp.c
+    exp_path = netuser/rhel9/exp.c
```

Comment:
I think can change the name to rhel/exp.c and use it for all disto, only use other name for specific, like RHEL8.

Reply:
Good suggestions, I have changed it, please help review it  again, thanks a lot.